foreman/foreman.spec

```diff
@@ -570,6 +560,7 @@ sed -i 's/:organizations_enabled: false/:organizations_enabled: true/' config/se
 export BUNDLER_EXT_NOSTRICT=1
 export BUNDLER_EXT_GROUPS="default assets"
 %{scl_rake} assets:precompile RAILS_ENV=production --trace
+%{scl_rake} webpack:compile --trace
```

Comment:
webpack isn't part of Foreman, so it wouldn't be distributed there. It'd just be packaged and set as a build dep.